Cargo.toml

```diff
@@ -27,10 +27,16 @@ defmt = ["dep:defmt"]
 # Supports serde
 serde = ["dep:serde"]
 
+borsh = ["dep:borsh"]
+
+schema = ["dep:schemars", "std"]
+
 [dependencies]
 num-traits = { version = "0.2.17", default-features = false, optional = true }
 defmt = { version = "0.3.5", optional = true }
 serde = { version = "1.0", optional = true, default-features = false}
+borsh = { version = "1.5.1", optional = true, features = ["unstable__schema"], default-features = false}
```

Comment:
Do you see a way to do this without relying on an unstable feature? I could see this break in the future

Reply:
> I could see this break in the future

I can add it as separate feature for BorshSchema?

Problem with borsh is next. Creators of Borsh already use this feature in production. And we too. BorshSchema does not requires nightly, it is just state of thing.

Why I do and have to add it? Because BorshShema is supported for u8/u16/u32/.../i8/... . So in order to replace any number with arbitrary, it to be supported. So what I say is that in our and NEAR chain scenarios, in order to replace Rust native numbers with arbitrary we need Schema.

Reply:
TLDR;

Assuming everyday will want to plug arbitrary int instead of existing int with highest possible fidelity, 
it means that if serde/schema/etc crate does something to native primitive int, 
than it must do same for arbitrary-int.

Arbitrary int should blend into like if they are native primitives.